storage/innobase/handler/ha_innodb.cc

```diff
@@ -11693,7 +11696,12 @@ inline MY_ATTRIBUTE((warn_unused_result)) int create_table_info_t::
   space_id_t space_id = 0;
   dd::Object_id dd_space_id = dd::INVALID_OBJECT_ID;
   ulint actual_n_cols;
-  fil_encryption_t keyring_encryption_option = FIL_ENCRYPTION_DEFAULT;
+
+  fil_encryption_t keyring_encryption_option =
+      Encryption::none_explicitly_specified(m_create_info->explicit_encryption,
+                                            m_create_info->encrypt_type.str)
+          ? FIL_ENCRYPTION_OFF
+          : FIL_ENCRYPTION_DEFAULT;
```

Comment:
Is this related to ENCRYPTION_KEY_ID change?

Reply:
no, in this PR I need to retain explicit ENCRYPTION='N' when ALTERing a table - as I explained in commit message.